src/sectionproperties/analysis/fea.py

```diff
@@ -198,27 +192,15 @@ def shear_load_vectors(
             h1 = -ixy * r + iyy * q
             h2 = -iyy * r - ixy * q
 
-            f_psi += (
-                gp[0]
-                * (
-                    nu
-                    / 2
-                    * np.transpose(np.transpose(b).dot(np.array([[d1], [d2]])))[0]
```

Comment:
Why take the first element `[0]`? The formula appears to be just B^T*[d1, d2].

Reply:
Will have a look into this now.

Reply:
All sorted.